TestProjects/UniversalGraphicsTest/Assets/Scenes/023_Lighting_Mixed_Indirect.unity

```diff
@@ -252,7 +252,7 @@ MonoBehaviour:
     PerPixelCorrectnessThreshold: 0.005
     PerPixelGammaThreshold: 0.003921569
     PerPixelAlphaThreshold: 0.003921569
-    AverageCorrectnessThreshold: 0.001
+    AverageCorrectnessThreshold: 0.0012
```

Comment:
curious of what we needed to bump the threshold with this PR.

Reply:
With Nicolaj we noticed that there is some noise in these baked lighting scenes (Maybe that is why they were disabled in first place, as I enabled them again)